packages/v4/patternfly-docs/content/design-guidelines/components/spinner/spinner.md

```diff
@@ -7,40 +7,47 @@ section: components
 
 Spinners are used to demonstrate data loading. Spinners are offered in different sizes that follow the sizing of our icons. These sizes include extra small, small, medium and large. Depending on the size of information you are loading and the space you are dealing with, we recommend using the spinner size that generally matches.
 
+
 <img src="./img/spinner-sizes.png" alt="Visual of different spinner sizes" width="436"/>
 
+## Placement
+
+Position the spinner in the center of the element that is loading. To ensure proper background contrast, use a white or light background
+
+<img src="./img/spinner-full-page.png" alt="Example of spinner in full page" width="990"/>
+
 ### When to use
 Use a spinner:
+- when the user is advancing to a new page or step where the structure has not been loaded in yet.
 - when the data you are dealing with is unknown in both quantity and shape.
 - when in progress of loading a screen that may not have any data, for example an [empty state screen or failed outcome](/components/empty-state). 
-- within a table view, where the spinner can take up the area of the table before it loads.
 - when the expected wait time is between 1-5 seconds. 
 
 ### When not to use
 Don't use a spinner:
 - to replace a [progress bar](/components/progress). 
-
 - if the loading process is less than 1 second. For experiences longer than 5 seconds, use a progress bar.
+- in components that load on top of the page, for example in cards or dropdowns. In this case use a [skeleton] (/components/skeleton) to reflect the structure of the data
 
 ### Spinner in context
 
 Spinners are centered within the container AND the viewport by default in all use cases. Although this is the standard position of it, the spinner may be repositioned if necessary for particular screens and screen sizes.
```

Comment:
I almost feel like we should omit this part if we're not going to be more specific "Although this is the standard position of it, the spinner may be repositioned if necessary for particular screens and screen sizes."

Like in what sense would it be repositioned? Wouldn't you still want it to be centered even on mobile?

Reply:
that was also already here but I can take it out. I agree that we should omit